LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2007 Mockito contributors
```

Comment:
It is correct date - 2007 for this project?

Reply:
Updated to 2020